save-plugins/fix-plugin/src/commonTest/kotlin/org/cqfn/save/plugins/fix/FixPluginTest.kt

```diff
@@ -116,19 +128,29 @@ class FixPluginTest {
         }
         val expectedFile = fs.createFile(tmpDir / "Test3Expected.java")
         fs.write(expectedFile) {
-            write("Expected file".encodeToByteArray())
+            if (isCurrentOsWindows()) {
+                write("Expected file \n".encodeToByteArray())
```

Comment:
So we still have a problem with this trailing space? Even if we are using cmd again, and not powershell?

Reply:
Yes, I found, that this problem initially appeared in CMD, and repeated in PowerShell for Windows.

In https://github.com/cqfn/save/pull/41, in tests of ProcessBuilder, I've encountered with same problems.
For now, I don't now some other workarounds

Reply:
OK, let's keep it like this. But please don't repeat line with `write` twice; introduce a viraibale `textPostfix` and call `write("Exceptec file$textPostfix")` to reduce code duplication